Code/Editor/EditorFramework/Dialogs/WindowCfgDlg.moc.h

```diff
@@ -28,6 +28,6 @@ private Q_SLOTS:
 
   ezUInt8 m_uiCurDesc = 0;
   ezWindowCreationDesc m_Descs[2];
-  bool m_bOverrideProjectDefault[2];
+  bool m_OverrideProjectDefault[2];

```

Comment:
I guess you are ignoring arrays ?

Reply:
Well, the rule is 'if it is a bool use the b prefix' not 'if it is a bool array'. If we do this for bool, will we do it for all other prefixes as well?

Reply:
yes